<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>488</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="RefreshButton">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FavoriteButton">
        <property name="text">
         <string>Добавить в избранное</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RemoveReplayButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="NickBox">
        <item>
         <property name="text">
          <string>Nick</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="MapBox">
        <item>
         <property name="text">
          <string>Карта</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="NationBox">
        <item>
         <property name="text">
          <string>Нация</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="TankBox">
        <item>
         <property name="text">
          <string>Класс техники</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="TypeTankBox">
        <item>
         <property name="text">
          <string>Танк</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="WinOrLose">
        <item>
         <property name="text">
          <string>Победа/проигришь</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="HeroOrNot">
        <item>
         <property name="text">
          <string>Герой</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Класс мастерства</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Все реплеи</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>75</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>14</number>
          </attribute>
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Игрок</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Карта</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Нация</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Техника</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Исход боя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Фраги</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Опыт</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кредиты</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Класс мастерства</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Выжил</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Избранные</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>75</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>14</number>
          </attribute>
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Игрок</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Карта</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Нация</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Техника</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Исход боя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Фраги</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Опыт</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кредиты</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Класс мастерства</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Выжил</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="settings"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settings">
   <property name="text">
    <string>Опции</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
